usaspending_api/recipient/tests/test_state.py

```diff
@@ -1,7 +1,9 @@
 # Stdlib imports
 import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
Can we move this file to fit within the following hierarchy/directory structure?
```
tests/unit/*
tests/integration/*
tests/data/*
```
Example: https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/broker/tests